### PR TITLE
True

### LINKED ISSUE #52: true
I want to do true

### PR DESCRIPTION
**Pull request by seanxwzhang**
Since my cat branch was declared by someone else, I made another reference for true
let me know if anything needs to be modified, thanks

Closes #52

### CODE REVIEW

**Comment by the-alex** [comment]
I understand there's not a lot of stuff you can do with "true", but we might want to make it more clear that while these are implemented as functions in bash, they are used like the logical boolean values you see in other languages like C++, python, and Java.

For instance, this script shows that `true` exits with `0` and `false` exits with `1`.
~~~bash
#!/usr/bin/env bash
# The value of $? is the exit status code of the last program/command to run.
true
echo $?   # Prints "0"
false
echo $?   # Prints "1"
~~~

#### Summary
* Explain what's going on in the first command `yes | true`, or use a better example and explain that.
* Make it more clear that this command is similar to the logical value `true` found in other languages.

**Comment by mterwill** [comment]
Please exclude changes to `cat.md`

**Comment by seanxwzhang** [comment]
I deleted cat.md and made modification as requested to true.md

File: _commands/basics/true.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+---
+
+true
+--
+
+`true` is a command that does nothing and return a true value(0).
```
Comment (the-alex) [inline _commands/basics/true.md]: If you change ...
~~~markdown
`true` is a command that does nothing and return a true value(0).
~~~
... to ...
~~~markdown
`true` is a command that returns the logical value of truth, which is `0`.
~~~
... then I think that takes care of everything for me.

If no one else has any comments, then I'll merge you in. Thank you so much!

**Comment by seanxwzhang** [comment]
I change the explanation as requested, thanks!

**Pull request merged**